src/styles/index.scss

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
Do we still need index.css for other GDS users, like WP sites without sass support? Or can I remove it?

If we need it, I guess it should be built based on index.scss..?

Reply:
I think it's good to have at least on non WP sites. With CSS variables that only leaves media queries that cant be modified

Reply:
Yeah, it would be good if it would build the index.css file from index.scss file.

Reply:
OK, I'll try to figure out how to build it.